internal/dockercompose/client.go

```diff
@@ -83,10 +92,14 @@ func (c *cmdDCClient) Down(ctx context.Context, configPath string, stdout, stder
 	return nil
 }
 
-func (c *cmdDCClient) StreamLogs(ctx context.Context, configPath string, serviceName model.TargetName) (io.ReadCloser, error) {
+func (c *cmdDCClient) StreamLogs(ctx context.Context, configPath []string, serviceName model.TargetName) (io.ReadCloser, error) {
 	// TODO(maia): --since time
 	// (may need to implement with `docker log <cID>` instead since `d-c log` doesn't support `--since`
-	args := []string{"-f", configPath, "logs", "-f", "-t", serviceName.String()}
+	var args []string
+	for _, config := range configPath {
+		args = append(args, "-f", config)
+	}
+	args = append(args, "logs", "-f", "-t", serviceName.String())
```

Comment:
Hmm what is this `"-f"` doing here?

Reply:
The first -f in the for loop specifies a file to compose as seen in the docker compose documentation: 
![image](https://user-images.githubusercontent.com/43042150/60600582-66f0ad80-9d7e-11e9-9eda-9c89a0732e04.png)


The second -f in the append method serves as an option of the logs command to "follow"  as seen in the docker compose logs documentation: 
![image](https://user-images.githubusercontent.com/43042150/60600479-3446b500-9d7e-11e9-9054-0cce4758aafb.png)